build.psm1

```diff
@@ -222,7 +222,15 @@ Fix steps:
             throw 'Win 10 SDK not found. Run Start-PSBootstrap or install Microsoft Windows 10 SDK from https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk'
         }
 
-        $vcVarsPath = (Get-Item(Join-Path -Path "$env:VS140COMNTOOLS" -ChildPath '../../vc')).FullName
+        # atlbase.h is included in at least one project
+        $vcPath = (Get-Item(Join-Path -Path "$env:VS140COMNTOOLS" -ChildPath '../../vc')).FullName
```

Comment:
More build diagnostics are good in my book.  I just wish this weren't tied specifically to VS 2015.  On at least one of my machines I have only VS 2017 installed.  @KirkMunro this is not your issue. Just an observation on this build script.